keyboards/handwired/dactyl_manuform/4x6/info.json

```
@@ -23,7 +23,8 @@
         "soft_serial_pin": "D0"
     },
     "processor": "atmega32u4",
-    "bootloader": "caterina",
+    "development_board": "promicro",
+    "pin_compatible": "promicro",
```

Processor and pin compatibility is set by dev board. 
```suggestion
    "development_board": "promicro",
```